src/scrabble.js

```diff
@@ -1,5 +1,82 @@
-function scrabble() {
-
+function scrabble(str) {
+  let counter = 0 // must be local to restart and redefine back to 0 when finished otherwise global scope will add it up
+  if (!str || str.length === 0) {
+    return 0
+  }
+  for (let i = 0; i < str.length; i++) {
```

Comment:
Interesting way to check the letters. It may look long but it's simple and works fine!